config/config.go

```diff
@@ -513,6 +514,7 @@ func NewConfig() *Config {
 	flag.DurationVar(&cfg.OpenPolicyAgentCleanerInterval, "open-policy-agent-cleaner-interval", openpolicyagent.DefaultCleanIdlePeriod, "Duration in seconds to wait before cleaning up unused opa instances")
 	flag.DurationVar(&cfg.OpenPolicyAgentStartupTimeout, "open-policy-agent-startup-timeout", openpolicyagent.DefaultOpaStartupTimeout, "Maximum duration in seconds to wait for the open policy agent to start up")
 	flag.Int64Var(&cfg.OpenPolicyAgentMaxRequestBodySize, "open-policy-agent-max-request-body-size", openpolicyagent.DefaultMaxRequestBodySize, "Maximum number of bytes from a http request body that are passed as input to the policy")
+	flag.Int64Var(&cfg.OpenPolicyAgentBodyReadBufferSize, "open-policy-agent-body-read-buffer-size", openpolicyagent.DefaultBodyBufferSize, "Read buffer size for the request body")
```

Comment:
Maybe we can make flag, option and constant names consistent,
e.g. `open-policy-agent-body-buffer-size` or `DefaultBodyReadBufferSize`.
Also maybe it makes sense to highlight that it is request body, e.g. similar to OpenPolicyAgentMaxRequestBodySize make it `OpenPolicyAgentRequestBodyBufferSize` (and `open-policy-agent-request-body-buffer-size`)

Reply:
Good points, will adjust...